src/test/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/integration/ClientsControllerIntTest.kt

```diff
@@ -496,6 +496,78 @@ class ClientsControllerIntTest : IntegrationTestBase() {
 
       clientRepository.delete(duplicatedClient)
     }
+
+    @Test
+    fun `verify duplicate, updated client`() {
```

Comment:
Not clear what this test is doing from the description. Each test case should only test one operation i.e. duplicate, not duplicate and update?

Reply:
removed unrelated test